### PR TITLE
Fix problem with quoted kernel arguments

### PR DESCRIPTION
**Pull request by javipolo**
/cc @tsorya 
/cc @donpenney

### CODE REVIEW

File: internal/bindata/prepSetupStateroot.sh [hunk 1]
```diff
@@ -42,7 +42,10 @@ function log_it {
 
 function build_kargs {
     jq -r '.spec.kernelArguments[]' "${img_mnt}/mco-currentconfig.json" \
-        | xargs --no-run-if-empty -I% echo -n "--karg-append % "
+        | while IFS= read -r line; do
```
Comment (romfreiman) [inline internal/bindata/prepSetupStateroot.sh:45]: shouldnt it move to golang?

Comment (javipolo) [reply internal/bindata/prepSetupStateroot.sh:45]: yes, this is a temporary fix to keep it working with 4.14 until we move to golang

**Pull request merged**